packages/block/test/block.spec.ts

```diff
@@ -205,7 +206,8 @@ describe('[Block]: block functions', () => {
     const blockRlp = hexToBytes(testdataPreLondonData.blocks[0].rlp as PrefixedHexString)
     const block = createBlockFromRLP(blockRlp, { common, freeze: false })
     await testTransactionValidation(block)
-    ;(block.transactions[0] as any).gasPrice = BigInt(0)
+    // @ts-expect-error -- Assigning to read-only property
+    block.transactions[0]['gasPrice'] = BigInt(0)
```

Comment:
Can't this then also do `block.transactions[0].gasPrice`? Instead of this bracket location? (also seen other occurences of this in the PR)